.travis.yml

```diff
@@ -31,12 +32,22 @@ script:
 - tsc --version
 - gts --version
 - gts check
-- node scripts/modifyConfigIfRCVersion.js
-- node scripts/updateResourceLink.js
-- node scripts/genAiKey.js
+- node scripts/updateConfig.js
 - vsce package
 - docker run -ti --rm -v $PWD:/mnt:ro dkhamsing/awesome_bot --allow-dupe --allow-redirect --skip-save-results `ls *.md`
 
+before_deploy:
+  # Set up git user name and tag this commit if nightly build
+  - if [[ $TRAVIS_EVENT_TYPE == "cron" ]]; then
+      git config --local user.name $USER_NAME &&

```

Comment:
Consider using a service account instead of a personal account.

Reply:
[https://github.com/vsciotci](https://github.com/vsciotci)

---
In reply to: [361107048](https://github.com/microsoft/vscode-iot-workbench/pull/884#discussion_r361107048) [](ancestors = 361107048)

Reply:
Done.